src/Strategy.sol

```diff
@@ -134,8 +278,238 @@ contract Strategy is BaseStrategy {
         virtual
         override
         returns (uint256)
+    {}
+
+    // ----------------- SUPPORT FUNCTIONS ----------
+
+    /**
+     * @notice
+     *  Takes the amount in `SYN` and sells it for `want` on solidly
+     * @dev
+     *  Total amount to sell in wei
+     **/
+    function _sellSynToWant(uint256 _amount) internal {
+        _checkAllowance(address(solidlyRouter), address(SYN), _amount);
+
+        IUniswapV2Router02.route[] memory routes = new IUniswapV2Router02.route[](1);
```

Comment:
We need to check if the amount of want we get back will be greater than 0. Otherwise, solidly will revert